zigpy/backups.py

```diff
@@ -108,8 +116,6 @@ def from_dict(cls, obj: dict[str, Any]) -> NetworkBackup:
                 obj["network_info"]["tx_power"] = None
                 version = 2
 
-            assert version == BACKUP_FORMAT_VERSION
```

Comment:
At least on production HA instances, this assertion wouldn’t be triggered, right? So it’s only an issue during development really.